Community/ideAlarm/automation/lib/python/configuration.py.example

```diff
@@ -38,7 +38,8 @@ idealarm_configuration = {
             'armAwayToggleSwitch': 'Toggle_Z1_Armed_Away',
             'armHomeToggleSwitch': 'Toggle_Z1_Armed_Home',
             'mainZone': True,
-            'canArmWithTrippedSensors': False
+            'canArmWithTrippedSensors': False,
+			'autoReset': False
```

Comment:
Tab indentation character used here and below. I believe it should be 4 space characters as elsewhere in the file.